mycore-restapi/src/main/java/org/mycore/restapi/MCRTransactionFilter.java

```diff
@@ -47,9 +47,10 @@ public void filter(ContainerRequestContext requestContext) {
             if (MCRTransactionManager.hasRollbackOnlyTransactions()) {
                 try {
                     MCRTransactionManager.rollbackTransactions();
-                } finally {
-                    throw new InternalServerErrorException("Transaction rollback was required.");
+                } catch(Exception exc) {
+                    LOGGER.error("Error rolling back transactions.", exc);
                 }
+                throw new InternalServerErrorException("Transaction rollback was required.");
```

Comment:
Exception suppresses exception catched above. You should instantiate Exception before try and add exception as suppressed in catch block